src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -116,7 +136,7 @@ define(function (require, exports, module) {
                                 disabled={this.props.disabled}
                                 document={this.props.document}
                                 layer={layer}
-                                childNodes={layerNode.children}/>
+                                hasChildren={!!layerNode.children}/>
```

Comment:
Not a problem, but I wonder why you decided to include this explicit prop when you've got the layer as a prop anyway (and this calculation could just be performed inside the layer).